src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -35,69 +36,94 @@ public interface DataAccessRequestDAO extends Transactional<DataAccessRequestDAO
    */
   @UseRowReducer(DataAccessRequestReducer.class)
   @SqlQuery(
-      "SELECT dd.dataset_id, dar.id, dar.reference_id, dar.collection_id, dar.parent_id, dar.draft, dar.user_id, dar.create_date, dar.sort_date, dar.submission_date, dar.update_date, "
-          + "  (regexp_replace(dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb AS data FROM data_access_request dar"
-          + "  LEFT JOIN dar_dataset dd on dd.reference_id = dar.reference_id "
-          + "  WHERE dar.draft != true "
-          + "  AND (LOWER(dar.data->>'status') != 'archived' OR dar.data->>'status' IS NULL)")
+      """
+          SELECT dd.dataset_id, dar.id, dar.reference_id, dar.collection_id, dar.parent_id, dar.user_id, dar.create_date, dar.sort_date, dar.submission_date, dar.update_date,
+            (regexp_replace(dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb AS data FROM data_access_request dar
+          LEFT JOIN dar_dataset dd on dd.reference_id = dar.reference_id 
+          WHERE dar.submission_date is not null
```

Comment:
switching from draft to submission date

Reply:
Thanks for these comments, they were very helpful!